languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/spelling.txt

```diff
@@ -1063,4 +1063,36 @@ morinom
 deadnaming
 morinommage
 morinommer
-Vivendi
\ No newline at end of file
+Vivendi
+Foy
+Colcott
+dim.
+lun.
+mar.
+mer.
+jeu.
+ven.
+sam.
+Plœuc
+offs
+pacemakeur
+pacemakeurs
+Virgin
+NRJ
```

Comment:
Please add abbreviations/acronyms to `ignore.txt`. Words in `ignore.txt` will be accepted by the speller, BUT not used to generate spelling suggestion for other misspelled words. Uncommon names and acronyms belong in there